### DISCUSSION TITLE
Allow logged in user to define and easily access personal shortcuts like google.com home page

### DISCUSSION THREAD

**Discussion by PJTewkesbury** [dailydotdev/daily#811] [Feature Requests]
Hi, I love the start page showing my news feed, but I do miss the access to my bookmarks bar. Also, I use Chrome and it's default new 
 tab page allows me to define frequently used websites as icons. It would be great if you could show the bookmarks bar, and allow me to define my personal shortcuts to my websites and display them on the page.

Thanks

Peter

**Comment by idoshamun** [comment]
Hi @PJTewkesbury,

Have you tried to use our shortcuts feature? https://docs.daily.dev/docs/getting-started/browser-extension-installation#adding-custom-shortcuts-to-dailydev

**Reply by stefano-vardanega** [reply]
How about making them customizable with a label?

**Reply by idoshamun** [reply]
This is currently out of scope for us, but we may consider it in the future 🙏
